code/modules/mining/ores_coins.dm

```diff
@@ -231,36 +232,43 @@
 /obj/item/weapon/coin/gold
 	cmineral = "gold"
 	icon_state = "coin_gold_heads"
+	materials = list(MAT_GOLD = 200)
 	value = 160
 
 /obj/item/weapon/coin/silver
 	cmineral = "silver"
 	icon_state = "coin_silver_heads"
+	materials = list(MAT_SILVER = 200)
 	value = 40
 
 /obj/item/weapon/coin/diamond
 	cmineral = "diamond"
 	icon_state = "coin_diamond_heads"
+	materials = list(MAT_DIAMOND = 200)

```

Comment:
Hang on, is there a method of putting a Diamond object into a protolathe/exosuit fab? Because if so, this is a bad idea, this thing is available at roundstart.